wordpress-mobile/WordPress-Android
# P2 people invite links unit test

Fixes #13456

This PR adds unit testing for 13456 issue introduced components.

## To test
- Check the unit tests pass
- I needed to add RoleUtilsWrapper.kt and SimpleDateFormatWrapper.kt to make it testable. Would be good to make some smoke testing just in case 😄 

PR submission checklist:
